internal/provider/data_source_packer_image_iteration.go

```diff
@@ -23,7 +23,7 @@ func dataSourcePackerImageIteration() *schema.Resource {
 		},
 		Schema: map[string]*schema.Schema{
 			// Required inputs
-			"bucket": {
+			"bucket_name": {
```

Comment:
It's totally fine to change this field directly while in private beta, but I wanted to mention this guide for any field name changes that happen after resource goes GA: https://www.terraform.io/docs/extend/best-practices/deprecations.html#provider-attribute-rename

Reply:
Thanks, yes, totally agreed to be careful with surprise deprecations 🙂